cmd/influx/main.go

```diff
@@ -83,56 +135,41 @@ func (c *cli) connect(cmd string) {
 	}
 }
 
-func (c *cli) parseCommand(cmd string) {
-	lcmd := strings.ToLower(cmd)
-	switch {
-	case strings.HasPrefix(lcmd, "exit"):
-		os.Exit(0)
-	case strings.HasPrefix(lcmd, "gopher"):
-		gopher()
-	case strings.HasPrefix(lcmd, "connect"):
-		c.connect(cmd)
-	case strings.HasPrefix(lcmd, "pretty"):
-		c.pretty = !c.pretty
-		if c.pretty {
-			fmt.Println("Pretty print enabled")
-		} else {
-			fmt.Println("Pretty print disabled")
-		}
-	case strings.HasPrefix(lcmd, "use"):
-		fmt.Println("We should use a database specified now... todo")
-	default:
-		c.executeQuery(cmd)
-	}
-}
-
 func (c *cli) executeQuery(query string) {
 	results, err := c.client.Query(client.Query{Command: query, Database: c.database})
 	if err != nil {
 		fmt.Printf("ERR: %s\n", err)
+		return
 	}
-	for _, r := range results.Results {
-		var i interface{}
-		if r.Err != nil {
-			i = r.Err
-		} else {
-			if len(r.Rows) == 0 {
-				continue
-			}
-			i = r.Rows
-		}
-		var data []byte
-		if c.pretty {
-			data, err = json.MarshalIndent(i, "", "    ")
-		} else {
-			data, err = json.Marshal(i)
-		}
-		if err != nil {
-			fmt.Printf("ERR: %s\n", err)
-			return
-		}
-		fmt.Fprintln(os.Stdout, string(data))
+	var data []byte
+	if c.pretty {
+		data, err = json.MarshalIndent(results, "", "    ")
+	} else {
+		data, err = json.Marshal(results)
 	}
+	if err != nil {
+		fmt.Printf("ERR: %s\n", err)
+		return
+	}
+	fmt.Fprintln(os.Stdout, string(data))
+}
+
+func help() {
+	fmt.Println(`Usage:
+        connect <host:port>   connect to another node
+        pretty                toggle pretty print
+        use <db_name>         set current databases
+        exit                  quit the influx shell
+
+        show databases        show database names
```

Comment:
@otoolep correct, nothing special.  I'm open to suggestions on what we think should be there.  I want to strike a balance between to much as I don't want to have this be in a state of flux if things change, but I do want it to give you some basic help.

At some point in the future, I'll leverage our system to actually do auto complete/suggestions.  but that is probably post 1.0.